api/rql/internal/primary/meta/valueSchema.go

```diff
@@ -0,0 +1,101 @@
+package meta
+
+import (
+	"fmt"
+	"strings"
+)
+
+/*
+ValueSchema represents a metadata value's schema. It is used to generate
```

Comment:
Looks like all the Schema and ValueSchema stuff in meta is not yet used.

Reply:
Yeah, my annoying habit of pushing stuff up before it is fully finished. I'll comment when the schema stuff's done.